From 532080ff7c05c403598ecd52f310e545f85d3226 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:41 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Update B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE to the current copy at
https://keys.openpgp.org/vks/v1/by-fingerprint/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE,
the URL the spec names for it (same certificate(s)).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE | 183 ++++++++++++++++-------
 1 file changed, 127 insertions(+), 56 deletions(-)

diff --git a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
index e28c60c..38e84af 100644
--- a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
+++ b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
@@ -5,6 +5,25 @@ Comment: Simon Josefsson <simon@josefsson.org>
 xjMEXJLOtBYJKwYBBAHaRw8BAQdACIcrZIvhrxDBkK9fV+QlTmXxo2naObDuGtw5
 8YaxlOvNJVNpbW9uIEpvc2Vmc3NvbiA8c2ltb25Aam9zZWZzc29uLm9yZz7ClgQT
 FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
+9jjFPAa+BQJq18y3BQkPYcIDAAoJENc89jjFPAa+570A/jRE5zhtTOrXPFTjdPvq
+tY/7qO+Yo78Jo96vVLZ9kw61AQC4wpDdf+o/vku2xskbzZNpoiSDZQzkzD6mnRQ5
+88noCsKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmnh9ZEFCQ5r6t0ACgkQ1zz2OMU8Br6FYgD/WVC+sl4a
+VCrnSU/q2b30PXMPLsnW1DeFrjHQnHuuQVMA/2GlQq5O+bqbV9fKJlVhmW7b4h38
+OrcIgBm80X5cSvYEwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AW
+IQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZ9F0JAUJDWRm8AAKCRDXPPY4xTwGvgbS
+AP4ke9wnwbqxvcvKQEod3GxjjB6Q2XzALaLLuo26TuvRhwD+Ke5ee3folbctlUDL
+Mft0wkgL2ZKa+D/xbk82fCWXvAbClgQTFggAPgIbAwULCQgHAgYVCAkKCwIEFgID
+AQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgIBQkLehFUAAoJENc8
+9jjFPAa+CboA+wUa06RD5e5VTCxvSWtPS75Wq2qBeYGZnf0jvUMxa2n4AP4xkUeA
+PPnNuMsTm2fsFCDIGaEM2Yn6Vb2huzzT1Fw/BMKWBBMWCAA+AhsDBQsJCAcCBhUI
+CQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmTyc40FCQl8
+aNkACgkQ1zz2OMU8Br7r3wD+J8GWC7Wt1KfiShDoTwTPVeF9mpkwDNs1bwOhDYFV
+JW0A/1tN1i20gTWPmkrz27T4kzFya+Hwfc/ee0ysRanZ4/MFwpYEExYIAD4CGwMF
+CwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+YzxUhgUJCHby0gAKCRDXPPY4xTwGvtUgAQCd8YrRkfJmSAaOkGrCzt2ilyCbGcQR
+yxQUoF3BhH6OowEAtesKlEoE43OeFp7RISnLsb4xKnsmZc0pJYGjfMjPFw3ClgQT
+FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
 9jjFPAa+BQJiOJ5MBQkGwpOYAAoJENc89jjFPAa+OVQBAIltTRSXhbbx9WSt7RNz
 M4u/1zwc4Ihpqoexu5w42xQoAP937CR37xoIUTyoKSvNeXnJSVkUOeRiCImuEGkD
 3OZrAsKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
@@ -24,71 +43,123 @@ fMYA/i0B5GsPglC7upw2a5qyTuSev7uaDLueBSV+mMq8cmAOwpYEExYIAD4WIQSx
 FgIDAQIeAQIXgAAKCRDXPPY4xTwGvpqwAQDPG0khoyBLa4gASbXlKAPxgFiflZy6
 EeOJHOUJkMYSrAEAjXUcy1wrw72CADiG5UUXrbBnRSNM3VwPxMvzQNAfcgDOMwRc
 ks91FgkrBgEEAdpHDwEBB0C8whXBx6ys1Ujww2q2T2Kp/PR+Uz2scHDkYEYPuA2o
-BMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nlsFCQbCkuYA
-CgkQ1zz2OMU8Br6mnAD9HvUcBNa1S1G/33f7uhqn9JDHnGGqFoA2iIkdtoOO5mIA
-/1n/ThshksmDE5Isuea3X92eqSga3+JeJlrVneMnqa0Lwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA9gUJBfBzgQAKCRDXPPY4xTwGvjLnAPoC
-iYpPmCcVKXoEmGBOFNcs+VzK/toEs6Vy0uFcaRJbiwEA/LLa/nZuEnjPRYIFCH7w
-41urLTtcC+SAW4KJyT8dQQ/CfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
-PAa+BQJf4imCBQkEjmUNAAoJENc89jjFPAa+dtIA/0DjkqcuxttFDfSROXHQTLde
-0hJebfQSxuXlPu9cYBrfAP0YCUov9QgHqfxH6/nbw6tbdUqKkB1mBfWefZdjr4a3
-D8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUcFCQN6SNIA
-CgkQ1zz2OMU8Br4MQQEAsh6/JxnmqDgJoTzzhgzmzqGuvw6bPlgaZrxcPCf2eQAB
-ALRP6cqR4rDB5heHMuUv5B1cy2b38WkwJRqraGRh0oQGwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIDQUJAjo8mAAKCRDXPPY4xTwGvu0SAQCK
-oSTVaP0+CerOAY5GTYs8A8tYiOo94pAWpA0FLOZzpwEA04K55KFRBu59exeb1yGH
-tXSJnQJCXrLnWOFpIwFQHADCfgQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
-BQJcks91AhsgBQkBHMQAAAoJENc89jjFPAa+5JIA/35ATQ4NJVxFiyZIxfMQQ5oL
-CW2GHbYDUKZehXmzXK6bAP9pVSpU3VDSZITqJpvjtDYSSWwIFOXxxKr8DUNx7BAp
-CM4zBFySz4EWCSsGAQQB2kcPAQEHQOxTCIOaeXAxI2hIX4HK9bQTpNVei708oNr1
-Klm8qCGKwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nl8F
-CQbCkt4AgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0Wi
+BMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmrXy7oFCQ9hwEIA
+CgkQ1zz2OMU8Br5sbQEAtfNvXfEwqtqhhvWuFk+0kCxHExBIBOWchdwaGAWBYzQB
+APuz2DuQ01MGEo2z3gsZ9Ug+4xublAdjgNRliVOB87AGwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCaeCW1wUJDmqLYgAKCRDXPPY4xTwGvrYzAQDB
+luzhj9iKANd4bw/D71qVWQMmrxo6djvlFvum3eatcwD/b0kW6aBYmLSmZdAYxdXy
+Y3jG+KhBTHA7z+sC84Q4wwDCfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJn0XQ1BQkNZGZAAAoJENc89jjFPAa+j7EBAKYaeN5v+U2CYATDi7seMlq1
+Hr9TsnN2Kwkp/SBE7maKAQDyQCUx6D39hLLli2VRnjVrBLpYBugmJel2+fzftWz+
+A8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmX9iCQFCQt6EK8A
+CgkQ1zz2OMU8Br6MQgEAnv4jEGbx2ZkkHn+UZy9RzlstNzh7bGcoRi50I7Mez4QB
+ANztitHTWz1tomTHJ/TFwDnZn3985Hza1hFpGUf8eV4Mwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYzxUlAUJCHbyHwAKCRDXPPY4xTwGvg2wAQDD
+NZ4Gzqn+VszD046QpEuU0lsWkx4LMxaURhWfNSQTUAD/e+r8Dt8Tlr4ZpTYfNiSz
+tsqvZlBVFzFBBp2Kw/AYDgjCfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJiOJ5bBQkGwpLmAAoJENc89jjFPAa+ppwA/R71HATWtUtRv993+7oap/SQ
+x5xhqhaANoiJHbaDjuZiAP9Z/04bIZLJgxOSLLnmt1/dnqkoGt/iXiZa1Z3jJ6mt
+C8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmD3wPYFCQXwc4EA
+CgkQ1zz2OMU8Br4y5wD6AomKT5gnFSl6BJhgThTXLPlcyv7aBLOlctLhXGkSW4sB
+APyy2v52bhJ4z0WCBQh+8ONbqy07XAvkgFuCick/HUEPwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IpggUJBI5lDQAKCRDXPPY4xTwGvnbSAP9A
+45KnLsbbRQ30kTlx0Ey3XtISXm30Esbl5T7vXGAa3wD9GAlKL/UIB6n8R+v528Or
+W3VKipAdZgX1nn2XY6+Gtw/CfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJezg1HBQkDekjSAAoJENc89jjFPAa+DEEBALIevycZ5qg4CaE884YM5s6h
+rr8Omz5YGma8XDwn9nkAAQC0T+nKkeKwweYXhzLlL+QdXMtm9/FpMCUaq2hkYdKE
+BsJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl2wSA0FCQI6PJgA
+CgkQ1zz2OMU8Br7tEgEAiqEk1Wj9PgnqzgGORk2LPAPLWIjqPeKQFqQNBSzmc6cB
+ANOCueShUQbufXsXm9chh7V0iZ0CQl6y51jhaSMBUBwAwn4EGBYIACYWIQSx0r0T
+db7LeEz0+MTXPPY4xTwGvgUCXJLPdQIbIAUJARzEAAAKCRDXPPY4xTwGvuSSAP9+
+QE0ODSVcRYsmSMXzEEOaCwlthh22A1CmXoV5s1yumwD/aVUqVN1Q0mSE6iab47Q2
+EklsCBTl8cSq/A1DcewQKQjOMwRcks+BFgkrBgEEAdpHDwEBB0DsUwiDmnlwMSNo
+SF+ByvW0E6TVXou9PKDa9SpZvKghisLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4
+xNc89jjFPAa+BQJq18u6BQkPYcA2AIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyH
+C50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwK
+OvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjX
+CrV4A0GmJA43+wD/TKZwRSwx6JmrwpF80WiRGmKmIoLx5e1u6lrrMIthYE0BAIO8
+SiBeExxy2Y+H+PK8BRu1VtrbPKP0mmKSTKIFiZcGwsA1BBgWCAAmAhsCFiEEsdK9
+E3W+y3hM9PjE1zz2OMU8Br4FAmngltcFCQ5qi1YAgQkQ1zz2OMU8Br52IAQZFggA
+HRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAf
+jlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8
+OGZwJcUi2NcKtXgDQaYkDrOwAPoCCB6QQKQ6XA49bCocw54Jqb8EH9FZB5nurPhB
+u6koKQEA3v1zK1NHND2t44vOXDKmYb8yThSP7KHuutpcy9EGNwXCwDUEGBYIACYC
+GwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZ9F0SgUJDWRmSQCBCRDXPPY4xTwG
+vnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5H
+RaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOp
+aPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOwUUBAO64fbZek6FPlRK0DrlWsrjCXuLi
+6PUxyzCAY6lG2nhUAQC6qobB9mkZlZ0qihy1x4JRtflqFcqqT9n7iUZkCDIiDcLA
+NQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgrBQkLehCqAIEJ
+ENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAK
+CRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6
+A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4ZxgD/bphBjTlsu2deIiNO
+IDOsZCtFfFMc8DwC/tMMUe8h61YA/2wdKhTw59PCZOIScbAuWxXtAlyVmDdg3P9i
+lyn1OpAGwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmM8VJkF
+CQh28hgAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0Wi
 BQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24Ue
-Yz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDhASAP9Kz68w
-Qt3DEyIW4ktaI6eeysvM93dHwgs+p8/j8kic7QD/bXTcLqs0J+tfTj/tN4CW6PcJ
-kx3dDJfNvGAY38vVUgzCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwG
-vgUCYPfA+gUJBfBzeQCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8v
+Yz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDmrpAP9II5n8
+OuXmWonPd7R5DTm5ViTVvPwOaISwtUxQn8uoCwEA7MZJ0/WtwlaWLOrGCET3Y2G3
+BHtgTgQp5Dpntfz2YQfCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwG
+vgUCYjieXwUJBsKS3gCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8v
 UXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7Jn
 hPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQO
-JVIBAJjPGgVSATrdkZIeKqhDNgwBp2sHHiiZW7N0wEw64tm4AQCie9uKfAIE2h+L
-eGVWZ5lfqoXNvhoE/8YWsNVdysQ7DMLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4
-xNc89jjFPAa+BQJf4imHBQkEjmUGAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyH
+EBIA/0rPrzBC3cMTIhbiS1ojp57Ky8z3d0fCCz6nz+PySJztAP9tdNwuqzQn619O
+P+03gJbo9wmTHd0Ml828YBjfy9VSDMLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4
+xNc89jjFPAa+BQJg98D6BQkF8HN5AIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyH
 C50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwK
 OvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjX
-CrV4A0GmJA5yiAEA6Ee/ptArOu2oGc2Okqf38MfN9q7SFTnzd5vutAI4Ib0BAKho
-dAz8j2HMUpxNVGjPHEwco7F1fkkR/78GJFsg+qADwsA1BBgWCAAmAhsCFiEEsdK9
-E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUwFCQN6SMsAgQkQ1zz2OMU8Br52IAQZFggA
+CrV4A0GmJA4lUgEAmM8aBVIBOt2Rkh4qqEM2DAGnawceKJlbs3TATDri2bgBAKJ7
+24p8AgTaH4t4ZVZnmV+qhc2+GgT/xhaw1V3KxDsMwsA1BBgWCAAmAhsCFiEEsdK9
+E3W+y3hM9PjE1zz2OMU8Br4FAl/iKYcFCQSOZQYAgQkQ1zz2OMU8Br52IAQZFggA
 HRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAf
 jlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8
-OGZwJcUi2NcKtXgDQaYkDuQiAPwKnKAbzegaSATxN1cdFia4m80uJNFHMQL679WS
-BG3FIAEA8uLgxGud6SqFgIaFR4wrzrIgzVWqHxDuu56fJSf/iAfCwDUEGBYIACYC
-GwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIEgUJAjo8kQCBCRDXPPY4xTwG
+OGZwJcUi2NcKtXgDQaYkDnKIAQDoR7+m0Cs67agZzY6Sp/fwx832rtIVOfN3m+60
+AjghvQEAqGh0DPyPYcxSnE1UaM8cTByjsXV+SRH/vwYkWyD6oAPCwDUEGBYIACYC
+GwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXs4NTAUJA3pIywCBCRDXPPY4xTwG
 vnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5H
 RaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOp
-aPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOte4A/2ju4S1HrcdnAVyd+BK5ieldpgOJ
-qKp9EFujf+zKuoKsAP4uq8+vBc6QmVri+ExSbqT/q5/pcUcBkxBVdda6G++yDMLA
-NQQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJcks+BAhsCBQkBHMQAAIEJ
+aPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQO5CIA/AqcoBvN6BpIBPE3Vx0WJribzS4k
+0UcxAvrv1ZIEbcUgAQDy4uDEa53pKoWAhoVHjCvOsiDNVaofEO67np8lJ/+IB8LA
+NQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJdsEgSBQkCOjyRAIEJ
 ENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAK
 CRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6
-A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4KLgEAuT/fZL+h/jy/QW9y
-fSi4BT56PGU7n/MoUzjOwG8flj4BAOCm+fqLhrM0SQzzjA0qdCrsTK/7rBSZ908t
-bNYOawgOzjgEXJLPahIKKwYBBAGXVQEFAQEHQDGVG6YNbqdljsD5UJ96T1WkTHo8
-mZ0nJg+3eU0cx3ocAwEIB8J+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAmI4nlQFCQbCkuoACgkQ1zz2OMU8Br7QXgEAhd81O+aan2sLslGjC749qnaq
-AHwan9Nxz8f+J9SzG3UBAPTMOR4aoC+u8KxUxkNU53Ccf+bgGHY4OJ0gTDLLpPIG
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA8QUJBfBzhwAK
-CRDXPPY4xTwGvtQ/APwP8XxDUEogiXWIatg4A8oap242CJQjPf9IrwETGf7ckAEA
-oF/XCoJNOsSKRGPgTnVA7vD8tNQpv6BZsk4WB3ShOAfCfgQYFggAJgIbDBYhBLHS
-vRN1vst4TPT4xNc89jjFPAa+BQJf4il7BQkEjmURAAoJENc89jjFPAa+410BAOK0
-ISCDC4Rd/LLUBMSfZsLIpI1yuFnDPp+OAc0Ec9tkAQDJ1Jz/XlxCwf/gYoqTLXkZ
-xGzd33QWjH2JKVZyKNF2CsJ+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAl7ODUAFCQN6SNYACgkQ1zz2OMU8Br7vEAD/S59d+oS7f8i38eCmIwcTwHX3
-faEWHd8alhQcTWZp/gYA/12a4TWmNzBAVeMiMeLbOiU+3SX1xnGzY8StU9gZV3UB
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIBgUJAjo8nAAK
-CRDXPPY4xTwGvuhAAQDvbSbrxBwmqvbEwJuprPVn9Nx/bwfWKfbS2Dqg/huOhAD9
-E1YBmxr5qiAbFJKP0O7Sj22HXX7LiALyf8eORehJPAjCfgQYFggAJhYhBLHSvRN1
-vst4TPT4xNc89jjFPAa+BQJcks9qAhsMBQkBHMQAAAoJENc89jjFPAa+4kMBAILG
-9rwrhcPSAvay4XDgJ8SqDHm8AsKA3E7zirxVq5X9AQCNlXB+RWh7yX61utkJrziB
-qQti/j3vHKX6c3yU4FLNCw==
-=KvZq
+A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA617gD/aO7hLUetx2cBXJ34
+ErmJ6V2mA4moqn0QW6N/7Mq6gqwA/i6rz68FzpCZWuL4TFJupP+rn+lxRwGTEFV1
+1rob77IMwsA1BBgWCAAmFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAlySz4ECGwIF
+CQEcxAAAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0Wi
+BQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24Ue
+Yz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDgouAQC5P99k
+v6H+PL9Bb3J9KLgFPno8ZTuf8yhTOM7Abx+WPgEA4Kb5+ouGszRJDPOMDSp0KuxM
+r/usFJn3Ty1s1g5rCA7OOARcks9qEgorBgEEAZdVAQUBAQdAMZUbpg1up2WOwPlQ
+n3pPVaRMejyZnScmD7d5TRzHehwDAQgHwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCatfLtwUJD2HATQAKCRDXPPY4xTwGvocHAP9g1zG1w10/lhb7
+ksrN9T8xnHTOZNibd27dk+X+leRCkQD+OWllYuM1N1ZA/a5Qjyv7pnq+hrxeqSg+
+cnm4ArlDQwbCfgQYFggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJp4JbX
+BQkOaottAAoJENc89jjFPAa+RNUA/2faQO/nFT06E+MlhlQdo/0chlQXC5TZMPTV
+vVBFwoLOAP9xLJK0ow5EjTzYJB4K810AL/Iv6PEOAEgA4cPTHVlbCcJ+BBgWCAAm
+AhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmfRdNIFCQ1kZugACgkQ1zz2OMU8
+Br7QzQEA89GrveQucdg0MyeJUE7NR2LZd4XfaJ2MRZZ+O/IglQ4A/jBn4AvEh6GK
+zdTDYHMZkTq21dDZD4N4Ik7dM+kippcMwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCZf2IMAUJC3oQxgAKCRDXPPY4xTwGvqYaAP46w6iK5f+eb1DA
+YSEL/kz+t+XdogkxJ8hZm1HD/foIegD6A7F2pVXdnDhsULUgvkxfD5GTJ9Uy5CSg
+I1ca/sceNALCfgQYFggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJjPFSP
+BQkIdvIlAAoJENc89jjFPAa+P30A/iZObTr1DpqdbdyQR+wWWc2Bsumo2PQEnr4V
+wnzxVmpCAP9+l84ozArXiyPBr6by61PZ0jv/SSEHnmLbjr71CL0XDMJ+BBgWCAAm
+AhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nlQFCQbCkuoACgkQ1zz2OMU8
+Br7QXgEAhd81O+aan2sLslGjC749qnaqAHwan9Nxz8f+J9SzG3UBAPTMOR4aoC+u
+8KxUxkNU53Ccf+bgGHY4OJ0gTDLLpPIGwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCYPfA8QUJBfBzhwAKCRDXPPY4xTwGvtQ/APwP8XxDUEogiXWI
+atg4A8oap242CJQjPf9IrwETGf7ckAEAoF/XCoJNOsSKRGPgTnVA7vD8tNQpv6BZ
+sk4WB3ShOAfCfgQYFggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJf4il7
+BQkEjmURAAoJENc89jjFPAa+410BAOK0ISCDC4Rd/LLUBMSfZsLIpI1yuFnDPp+O
+Ac0Ec9tkAQDJ1Jz/XlxCwf/gYoqTLXkZxGzd33QWjH2JKVZyKNF2CsJ+BBgWCAAm
+AhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUAFCQN6SNYACgkQ1zz2OMU8
+Br7vEAD/S59d+oS7f8i38eCmIwcTwHX3faEWHd8alhQcTWZp/gYA/12a4TWmNzBA
+VeMiMeLbOiU+3SX1xnGzY8StU9gZV3UBwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCXbBIBgUJAjo8nAAKCRDXPPY4xTwGvuhAAQDvbSbrxBwmqvbE
+wJuprPVn9Nx/bwfWKfbS2Dqg/huOhAD9E1YBmxr5qiAbFJKP0O7Sj22HXX7LiALy
+f8eORehJPAjCfgQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJcks9qAhsM
+BQkBHMQAAAoJENc89jjFPAa+4kMBAILG9rwrhcPSAvay4XDgJ8SqDHm8AsKA3E7z
+irxVq5X9AQCNlXB+RWh7yX61utkJrziBqQti/j3vHKX6c3yU4FLNCw==
+=d6lo
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0


